txcheck/sigcheck.go

```diff
@@ -0,0 +1,72 @@
+package txcheck
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	coreData "github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/builders"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+)
+
+// SigVerifier defines the methods available for a signature verifier
+type SigVerifier interface {
```

Comment:
not really needed, `builders.TxSigVerifier` can be used instead

Reply:
done